Fix view page from template allowed in Space

yurabakhtin opened the pull request:
Issue: https://github.com/humhub/custom-pages/issues/251

luke- reviewed (changes requested):
@yurabakhtin can you please check the tests?

yurabakhtin commented:
@luke- 

> can you please check the tests?

Fixed in the commit https://github.com/humhub/custom-pages/pull/252/commits/03aa3be5635b5815e38bd7756234d06c767ec9d2, it seems only my local php 8.1.5 has the issue, because in first commit here https://github.com/humhub/custom-pages/pull/252/commits/6e2cc0d961b59ae6a917d31ac0cf58d520ea0fe3#diff-c37214cc53a8607c16e3cfe69c019b5ccc82e567153098beb4ccf3b5b4cfed90R47 I added the `mixed` in order to fix the error:

`During inheritance of ArrayAccess: Uncaught yii\base\ErrorException: Return type of humhub\modules\custom_pages\modules\template\models\AssetVariable::offsetGet($offset) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in ...\AssetVariable.php:47`

![php8 1-error](https://user-images.githubusercontent.com/6995524/200567972-1986604a-99b1-43f0-9d0d-c7c3b45d5198.png)

but if the github php 8.1 has no issue we may keep this version, but probably some servers may have the same bug.

luke- commented:
@yurabakhtin Hmm, strange. Lets keep the current version and see if there are any further probs.
